python/prairielearn.py

```diff
@@ -1547,8 +1552,8 @@ def latex_from_2darray(A, presentation_type="f", digits=2):
         .splitlines()
     )
     rv = [r"\begin{bmatrix}"]
-    rv += ["  " + " & ".join(line.split()) + r"\\" for line in lines]
-    rv += [r"\end{bmatrix}"]
+    rv.extend("  " + " & ".join(line.split()) + r"\\" for line in lines)
+    rv.append(r"\end{bmatrix}")
```

Comment:
Nit: why not just declare `rv` as a list with three element?

Reply:
It's not 3 elements, the middle one is a list comprehension.

Reply:
Indeed - I missed that! Thanks for pointing that out.